### DISCUSSION TITLE
Cannot sign in

### DISCUSSION THREAD

**Discussion by ghost** [community/community#138264] [Other Feature Feedback, Questions, & Ideas] [answered]
![image](https://github.com/user-attachments/assets/6897ebf0-717e-4927-b2a1-9b016cd8ff09)

I forgot my password on my original email and when I try to reset it get an error about it not being a verified primary email. How do I fix this and sign in?

**Comment by queenofcorgis** [comment, marked as answer]
Hi @DemDoidDeedoid ! The best place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact). The community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours. We appreciate your understanding! 

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion.

Thank you!

_Please note: You are more likely to get a useful response if you are posting your question in the applicable category. The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them._
